events/smart-descriptions.json

```diff
@@ -1,4 +1,337 @@
+                },
+                {
+                    "field": "event.reason",
+                    "value": "Sent"
+                }
+            ],
+            "relationships": [
+                {
+                    "source": "action.properties.relay",
+                    "target": "email.to",
+                    "type": "accepted a message form the following relay"
+                }
+            ],
+            "value": "{email.to} accepted a message form the following relay {action.properties.relay}"
+        },
+	     {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "even.reason",
+                    "value": "sent"
+                }
+            ],
+            "relationships": [],
+            "value": "A message to {email.to} was {action.properties.stat}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "spam"
+                }
+            ],
+            "relationships": [],
+            "value": "A spam activity has been detected from source IP {source.ip}: {event.message}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "virus"
+                },
+                {
+                    "field": "file.type",
+                    "value": "file"
+                },
+                {
+                    "field": "action.properties.event_status",
+                    "value": "CLEAN"
+                }
+            ],
+            "relationships": [],
+            "value": "A {file.type} nammed {file.name} with the related checksum: {file.hash.sha256} was concidered as {action.properties.event_status}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "virus"
+                },
+                {
+                    "field": "file.type",
+                    "value": "file"
+                }
+            ],
+            "relationships": [],
+            "value": "A {file.type} nammed {file.name} with the related checksum: {file.hash.sha256} with no specific status"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "virus"
+                }
+            ],
+            "relationships": [],
+            "value": "A virus activity has been detected: {event.message}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "encrypt"
+                }
+            ],
+            "relationships": [],
+            "value": "An encryption activity has been detected: {event.message}"
+        }
```

Comment:
Not sure these descriptions are adding value since `event.message` is the default. Do you have examples of such messages ?